templates/python/custom-copilot-rag-customize/src/requirements.txt

```diff
@@ -0,0 +1,5 @@
+python-dotenv
+aiohttp
+azure-search
```

Comment:
do we need the azure-search related packages?

Reply:
no we dont. will remove it